src/main/java/org/spongepowered/common/config/category/TimingsCategory.java

```diff
@@ -33,22 +33,22 @@
 @ConfigSerializable
 public class TimingsCategory extends ConfigCategory {
 
-    @Setting
+    @Setting(comment = "???")
     private boolean verbose = false;
 
-    @Setting
+    @Setting(comment = "???")
     private boolean enabled = true;
 
-    @Setting("server-name-privacy")
+    @Setting(value = "server-name-privacy", comment = "???")
     private boolean serverNamePrivacy = false;
 
-    @Setting("hidden-config-entries")
+    @Setting(value = "hidden-config-entries", comment = "???")
```

Comment:
allows you to hide specific config entries from showing up on the webviewer, e.g. not leak your sql creds

Reply:
Phit is correct.
Just a small note: these config entries are skipped and are never sent to the webviewer (aka they don't leave your server).
Additionally the sponge.sql section is **always** ignored and is never sent to the webviewer (regardless of this option).